unified-runtime/test/conformance/exp_command_buffer/native-command/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2024 Intel Corporation
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+if (UR_BUILD_ADAPTER_CUDA)
+    target_sources(test-exp_command_buffer PRIVATE "append_native_cuda.cpp")
+    target_link_libraries(test-exp_command_buffer PRIVATE cudadrv)
+endif()
+
+if (UR_BUILD_ADAPTER_HIP)
+    target_sources(test-exp_command_buffer PRIVATE "append_native_hip.cpp")
+    target_link_libraries(test-exp_command_buffer PRIVATE rocmdrv)
+endif()
+
+if (UR_BUILD_ADAPTER_L0 OR UR_BUILD_ADAPTER_L0_v2)
+    target_sources(test-exp_command_buffer PRIVATE "append_native_L0.cpp")
+    target_link_libraries(test-exp_command_buffer PRIVATE LevelZeroLoader)
+endif()
+
+if (UR_BUILD_ADAPTER_OPENCL)
+    target_sources(test-exp_command_buffer PRIVATE "append_native_opencl.cpp")
+    target_include_directories(test-exp_command_buffer PRIVATE ${OCL_INCLUDE_DIR})
```

Comment:
then you can remove this since the include directory will be transitive from linking against `OpenCLICCDLoader`

```suggestion
```